src/components/buttons/Buttons.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Buttons = ({ handleClick, changeName }) => (
```

Comment:
handleClickWhat?
в большом коде не будет понятно какой именно клик хендлит эта функция
```suggestion
export const Buttons = ({ toggleHandler, changeName }) => (
```